Dockerfile

```diff
@@ -1,8 +1,7 @@
 FROM node:lts-alpine
-RUN mkdir -p /usr/src/fosscord-api
 WORKDIR /usr/src/fosscord-api
-COPY package.json /usr/src/fosscord-api
```

Comment:
@DiegoMagdaleno did two commands as the first one caches and install all dependencies and the second copies the source files.
This is needed as docker caches the docker image by changes and doesn't need to reinstall all dependencies if you only change a src file

Reply:
I rolled back the change